fdbserver/RestoreLoader.actor.cpp

```diff
@@ -471,11 +480,10 @@ void _parseSerializedMutation(VersionedMutationsMap* pkvOps, SerializedMutationL
 }
 
 // Parsing the data blocks in a range file
-ACTOR static Future<Void> _parseRangeFileToMutationsOnLoader(VersionedMutationsMap* pkvOps,
-                                                             Reference<IBackupContainer> bc, Version version,
-                                                             std::string fileName, int64_t readOffset, int64_t readLen,
-                                                             KeyRange restoreRange) {
-	state VersionedMutationsMap& kvOps = *pkvOps;
+ACTOR static Future<Void> _parseRangeFileToMutationsOnLoader(
+    std::map<LoadingParam, VersionedMutationsMap>::iterator kvOpsIter, Reference<IBackupContainer> bc, Version version,
+    std::string fileName, int64_t readOffset, int64_t readLen, KeyRange restoreRange) {
```

Comment:
The API can be simplified by taking `LoadingParam`, `VersionedMutationsMap`, instead of the iterator and some fields within the iterator.

Reply:
Not sure if the pointer to the VersionedMutationsMap is stable when the map changes. 
The map::iterator is stable when inserting or deleting elements into the map.